Tests/TestHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace Azine\EmailBundle\Tests;
+
+use Azine\EmailBundle\Entity\SentEmail;
+use Azine\EmailBundle\Services\AzineTemplateProvider;
+use Doctrine\ORM\EntityManager;
+
+class TestHelper
+{
+    const TEST_EMAIL = 'test@mail.com';
```

Comment:
mail.com is a real domain. For testing purposes "example.com" should be used. I myself have not followed that rule in the past... :-/ I'll have to fix that in other places in my code too. :-)